cmake/findDependencies.cmake

```diff
@@ -30,6 +28,8 @@ else()
     list(APPEND CMAKE_PREFIX_PATH ${CMAKE_BINARY_DIR})
 endif()
 
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/")
```

Comment:
Why did you change the order of this line? I think that it was before the Conan section on purpose. Nonetheless, I would need to check this out with a computer. Right now I am reviewing the code from a tablet.

Reply:
Because the hand-made cmake/FindBrotli.cmake was being used, and that one doesn't know about Conan's install paths so it was not finding anything...